themes/stanford_basic/stanford_basic.theme

```diff
@@ -145,6 +145,12 @@ function stanford_basic_preprocess_node(&$variables) {
       $variables['attributes']['class'][] = 'add-more-space-to-top';
     }
   }
+  if (
+    $variables['page'] &&
+    $node_bundle == 'stanford_opportunity'
+  ) {
+    $variables['attributes']['class'][] = Html::cleanCssIdentifier('node--' . $node_bundle);
```

Comment:
@pookmish How is this?

Reply:
So close. Now you can remove the conditional check in the `if` statement above (`$node_bundle == ...`). That will apply the dynamic classes to all node pages.

Reply:
@pookmish Ok. now try.